reduce_test.go

```diff
@@ -16,6 +16,37 @@ var largeEpsilon = 0.001
 
 func floatMostlyEquals(a float64, b float64) bool {
 	return (a-b) < largeEpsilon && (b-a) < largeEpsilon
+
+}
+
+func TestMaxAggregateInterpolateWhenInsufficientData(t *testing.T) {
```

Comment:
I can see how you chose this name for the test but I don't believe it fits. The specific `InterpolateWhenInsufficientData` is meant to show that the percentile related tests are operating on less than a full data set. The min and max functions are not able to have "insufficient data" and do not interpolate like the percentile functions.

Would you change these to `TestMax` and `TestMin` so that future readers know this?

Reply:
Thanks for your suggestion.I will fix it.

Reply:
Done. Please take a look again.